bundles/org.openhab.binding.fmiweather/src/main/java/org/openhab/binding/fmiweather/internal/AbstractWeatherHandler.java

```diff
@@ -94,7 +95,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     logger.trace("REFRESH received. Delaying by {} ms to avoid throttle excessive REFRESH",
                             delayRemainingMillis);
                 }
-                if (prevFuture == newFuture) {
```

Comment:
Adding this comment again as github seemingly lost it after Force push.

This has been intentionally comparing by identity.

Reply:
Addressed as well, now I see the comment. :wink:

Reply:
Thanks! Good to have the comment  👌